src/schema/landingPage/resolvers.ts

```diff
@@ -7,27 +7,69 @@ const normalizeLandingPage = (collection) => {
   let normalizedLandingPage = normalizeKeys(collection);
   const normalizedKeys = [
     'title',
+
+    // BEGIN deprecated fields
     'description',
+    'socialMediaImage',
     'titleAndDescriptionColor',
     'buttonText',
     'buttonUrl',
     'heroBackgroundImage',
     'heroBackgroundImageMobile',
     'heroBackgroundImageColor',
     'heroTopLayerImage',
+    // END deprecated fields
+
     'keywords',
-    'socialMediaImage',
     'metaInformation',
     'pageTitle',
+    {
+      bottomBanner: [
+        'title',
+        'description',
+        'titleAndDescriptionColor',
+        'buttonText',
+        'buttonUrl',
+        'heroBackgroundImage',
+        'heroBackgroundImageMobile',
+        'heroBackgroundImageColor',
+        'heroTopLayerImage',
+        'socialMediaImage',
+      ],
+    },
+    {
+      topBanner: [
+        'title',
+        'description',
+        'titleAndDescriptionColor',
+        'buttonText',
+        'buttonUrl',
+        'heroBackgroundImage',
+        'heroBackgroundImageMobile',
+        'heroBackgroundImageColor',
+        'heroTopLayerImage',
+        'socialMediaImage',
+      ],
+    },
   ];
 
   normalizedKeys.forEach((item) => {
-    normalizedLandingPage = normalizeLocalizedObject(
-      normalizedLandingPage,
-      item
-    );
+    if (typeof item === 'string') {
+      normalizedLandingPage = normalizeLocalizedObject(
+        normalizedLandingPage,
+        item
+      );
+    } else if (typeof item === 'object') {
+      Object.entries(item).forEach(([nestedFieldName, nestedFields]) => {
+        nestedFields.forEach((localizeObject) => {
+          normalizedLandingPage[nestedFieldName] = normalizeLocalizedObject(
+            normalizedLandingPage[nestedFieldName],
+            localizeObject
+          );
+        });
+      });
+    }
```

Comment:
we could avoid this if we implemented helper function that goes recursively through all the fields

Reply:
Can be confusing if there is lot of synonyms in codebase, preferably avoid them.

Reply:
That would be a good idea in case API returns more depth data. However, as I mentioned above, the Wagtail CMS API is now deprecated and I don't think we will have any deeper data anytime soon. I think making a complex helper function is not necessary at the moment. Instead, I'll try to change the API to avoid returning Localized Objects in nested attributes

Reply:
👍 ok